org.phoenicis.playonlinux.json

```diff
@@ -0,0 +1,80 @@
+{
+  "app-id": "org.phoenicis.playonlinux",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "add-extensions": {
+    "org.freedesktop.Platform.Compat.i386": {
+      "directory": "lib/i386-linux-gnu",
+      "version": "18.08"
+    },
+    "org.freedesktop.Platform.Compat.i386.Debug": {
+      "directory": "lib/debug/lib/i386-linux-gnu",
+      "version": "18.08",
+      "no-autodownload": true
+    },
+    "org.freedesktop.Platform.GL32": {
+      "directory": "lib/i386-linux-gnu/GL",
+      "version": "18.08",
+      "subdirectories": true,
+      "no-autodownload": true,
+      "autodelete": false,
+      "add-ld-path": "lib",
+      "merge-dirs": "vulkan/icd.d;glvnd/egl_vendor.d",
+      "download-if": "active-gl-driver",
+      "enable-if": "active-gl-driver"
+    }
+  },
+  "command": "phoenicis-playonlinux",
+  "finish-args": [
+    "--filesystem=home",
```

Comment:
Is the full home directory access really needed?

Reply:
We use `~/.Phoenicis` only. Is it possible to restrict it to that?

Reply:
Yup. :-) You can use `--persist=.Phoenicis`.

Reply:
By the way, there was also a `.org.phoenicis.javafx.JavaFXApplication` directory created in home by PoL, with webview/localstorage subdirectory and webview/.lock file inside.

Reply:
I've just tried this but it doesn't seem to work. I think I must adjust the directory (Java property) we are using somehow. Where is the directory mapped to inside the runtime?

Reply:
@plata I just tried replacing the `--filesystem=home` with `--persist=.Phoenicis` and everything works absolutely fine without any further modifications.

By the way, I would also recommend you adding `--filesystem=xdg-download` to allow installing apps from local installers and `--filesystem=xdg-documents` to allow accessing user's documents.

Reply:
It works but I don't think it is using `~/.Phoenicis`. Try the following: Run Phoenicis PlayOnLinux without flatpak and install an app. Then, run Phoenicis PlayOnLinux with flatpak. You will not see the app. We don't want the apps to reside in the runtime because Phoenicis is just a mean to install them (they do not belong to it). You might want to compare it e.g. to a text editor where you can only save files inside the flatpak.

Reply:
One of the major points of Flatpak is that it makes apps (including their file system access) sandboxed. The text editor can use a portal to open/save files, it doesn't need full file system access. Anyway, if you really want the _.Phoenicis_ directory to be unsandboxed, `--filesystem=.Phoenicis` should work fine.

Reply:
Ok, I will try that.

Reply:
It works.

I've noticed that the behavior is a bit strange though: You can either select a file or drag and drop it. Drag and drop accepts any file (even from directories which cannot be accessed). The file cannot be opened but that's not really obvious because the drag and drop accepts it first.